base/linalg/generic.jl

```diff
@@ -514,11 +539,29 @@ julia> norm(v)
 julia> norm(v, Inf)
 6.0
 ```
+"""
+norm(x::AbstractVector, p::Real=2) = vecnorm(x, p)
+
+"""
+    norm(A::AbstractMatrix, p::Real=2)
 
 For matrices, the matrix norm induced by the vector `p`-norm is used, where valid values of
```

Comment:
As I said I'm not familiar with this, but reading the [Wikipedia page](https://en.wikipedia.org/wiki/Matrix_norm#Matrix_norms_induced_by_vector_norms) I wouldn't think "matrix norm" is more precise than "operator norm", since there are several definitions for matrix norms.